packages/schematics/angular/migrations/migration-collection.json

```diff
@@ -29,6 +29,11 @@
       "version": "7.0.3",
       "factory": "./update-7/index#updateDevkitBuildNgPackagr",
       "description": "Update an Angular CLI project to version 7."
+    },
+    "migration-07": {
+      "version": "7.4.0",
```

Comment:
I chose 7.4.0 under the assumption that there will be a version of the cli released with that version number. Is that accurate? Otherwise I can make it 8.0.0